src/containers/AudioUploader/components/AudioPlayer.jsx

```diff
@@ -12,13 +12,13 @@ import { Field } from '../../../components/Fields';
 
 class AudioPlayer extends React.Component {
   componentWillReceiveProps(props) {
-    if (this.props.audio.url !== props.audio.url) {
+    if (this.props.filepath !== props.filepath) {
       this.audioPlayer.pause();
       this.audioPlayer.load();
     }
   }
   render() {
-    const { audio, noBorder, bigText } = this.props;
+    const { audio, noBorder, bigText, filepath } = this.props;
     return (
       <Field noBorder={noBorder} bigText={bigText}>
```

Comment:
Med `Field` så inkluderer den css som gir `audio` luft ovenfor og nedenfor og jeg antar at `Field` plasserer audio spilleren responsivt. Uten den måtte jeg skrive css i tillegg for å plassere den pent på siden.